soccer/Referee.cpp

```diff
@@ -108,32 +107,32 @@ static const int KickVerifyTime_ms = 250;
 // the ref halts/stops, make this false
 static const bool CancelBallPlaceOnHalt = true;
 
-NewRefereeModule::NewRefereeModule(Context* const context, bool isBlue)
+Referee::Referee(Context* const context, bool isBlue)
     : stage(NORMAL_FIRST_HALF_PRE),
       command(HALT),
       _running(false),
       _context(context),
       _blueTeam(isBlue) {}
 
-NewRefereeModule::~NewRefereeModule() { this->stop(); }
+Referee::~Referee() { stop(); }
 
-void NewRefereeModule::stop() {
+void Referee::stop() {
     _running = false;
     wait();
 }
 
-void NewRefereeModule::getPackets(std::vector<NewRefereePacket*>& packets) {
+void Referee::getPackets(std::vector<RefereePacket*>& packets) {
     _mutex.lock();
     packets = _packets;
     _packets.clear();
     _mutex.unlock();
 }
 
-void NewRefereeModule::run() {
+void Referee::run() {
     QUdpSocket socket;
```

Comment:
@kylestach Did we want to switch to boost::asio instead of qt?

Reply:
yeah I think we did, but I think that should be a separate PR as it's like a different concern?

Reply:
That works, I just vaguely remembered something about that and just wanted to make sure.

Reply:
Yeah, that can happen later if we want. I'm just annoyed by how Qt tries to be literally everything (QMutex, QString, QUdpSocket, ...).